chrome/extension/background.js

```diff
@@ -1,29 +1,5 @@
-const bluebird = require('bluebird');
-const _ = require('lodash');
-
-global.Promise = bluebird;
-
-function promisifier(method) {
-  // return a function
-  return function promisified(...args) {
-    // which returns a promise
-    return new Promise(resolve => {
-      args.push(resolve);
-      method.apply(this, args);
-    });
-  };
-}
-
-function promisifyAll(obj, list) {
-  list.forEach(api => bluebird.promisifyAll(obj[api], { promisifier }));
-}
-
-// let chrome extension api support Promise
-promisifyAll(chrome, ['tabs', 'windows', 'browserAction']);
-promisifyAll(chrome.storage, ['local']);
-
 const onIconClicked = () => {
```

Comment:
may be we can make it more meaningful like:
`onIconClicked` -> `onYoroiIconClicked`